test/integration/container/multimaster_test.go

```diff
@@ -293,15 +303,15 @@ func TestMultimasterSetup(t *testing.T) {
 			for _, apiServerArg := range expectedApiServerArgs {
 				log.Infof("Checking api server arg (%s) on node%d", apiServerArg, i)
 				run(t, "footloose",
-					"-c", "../../../examples/footloose/centos7/docker/multimaster.yaml",
+					"-c", "../../../examples/footloose/"+node_os+"/docker/multimaster.yaml",
 					"ssh", fmt.Sprintf("root@node%d", i), fmt.Sprintf("ps -ef | grep -v 'ps -ef' | grep kube-apiserver | grep %s", apiServerArg))
 			}
 		}
 	}
 
 	if !t.Failed() { // Otherwise leave the footloose "VMs" & config files around for debugging purposes.
 		// Clean up:
-		defer runIgnoreError(t, "footloose", "delete", "-c", "../../../examples/footloose/centos7/docker/multimaster.yaml")
+		defer runIgnoreError(t, "footloose", "delete", "-c", "../../../examples/footloose/"+node_os+"/docker/multimaster.yaml")
```

Comment:
Ditto

Reply:
Odd, that is exactly what I have in my local file and the file doesn't have any changes since I pushed it up (save for one I added regarding your comment alone).  I'm baffled why that is...